ios/Exponent/Kernel/ReactAppManager/EXReactAppManager.mm

```diff
@@ -246,13 +246,6 @@ - (NSString *)versionedString: (NSString *)string
   return [EXVersions versionedString:string withPrefix:_versionSymbolPrefix];
 }
 
-- (NSString *)escapedResourceName:(NSString *)string
```

Comment:
This was causing additional unnecessary escapes in the scoped directory.

![Screenshot 2023-12-01 at 17 55 05](https://github.com/expo/expo/assets/1714764/86d591c8-babe-4748-87ff-88913b9638a3)

Reply:
Should we automatically migrate those directories for users? As a result of this change, all projects will get new (correct) cache and document directories, thus everything that people saved there will be lost. In my opinion migrating them for users would only increase maintenance cost on our side and it doesn't seem to be a big problem if the projects will start from scratch.

Reply:
>  IIRC we stopped using scoping for production apps in most modules a long time ago.

correct

> if this affects only Expo Go, I don't think we need to migrate the directories

I agree with @ide - given that this only impacts Expo Go we don't need to bother with migrating the data. Our reasoning is that Expo Go is a sandbox environment and as such it shouldn't be depended on for real applications where the single source of data lives within the Expo Go app.